org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/thingTranslator.js

```diff
@@ -16,12 +16,67 @@ function validateArgumentType(arg, argName, expectedType) {
     }
 }
 
+// Helper method to convert the device schema to the translator schema.
+function deviceSchemaToTranslatorSchema(deviceSchema) {
+
+    // Quirks:
+    // - Nest does not have an external temperature field, so returning the ambient temperature instead
```

Comment:
> // - Nest does not have an external temperature field, so returning the ambient temperature instead

I believe it should be valid to leave out properties if they're not marked as 'required' in the schema. I'm not sure I see value in including an externalTemperature if Nest doesn't support that.